src/pages/LoginForm.js

```diff
@@ -0,0 +1,170 @@
+        boxShadow="lg"
+      >
+        {signUp ? (
+          <Box p={2}>
+            {/* Sign In Page */}
+            <Box textAlign="center">
+              <Heading>Get started</Heading>
+            </Box>
+            <Box my={4} textAlign="left">
+              <form onSubmit={handleSubmit}>
+                {error ? <ErrorMessage message={error} /> : null}
+                <FormControl isRequired>
+                  <FormLabel>Email</FormLabel>
+                  <Input
+                    type="email"
+                    placeholder="someone@somewhere.com"
+                    name="email"
+                    onChange={handleChange}
+                  />
+                </FormControl>
+                <FormControl mt={6} isRequired>
+                  <FormLabel>Password</FormLabel>
+                  <Input
+                    type="password"
+                    placeholder="*******"
+                    name="password"
+                    onChange={handleChange}
+                  />
+                </FormControl>
+                <FormControl mt={6} isRequired>
+                  <FormLabel>Confirm Password</FormLabel>
+                  <Input
+                    type="password"
+                    placeholder="*******"
+                    name="confirmPassword"
+                    onChange={handleChange}
+                  />
+                </FormControl>
+                <Button
+                  width="full"
+                  mt={4}
+                  type="submit"
+                  variantColor="teal"
+                  variant="outline"
+                >
+                  {isLoading ? (
+                    <CircularProgress
+                      isIndeterminate
+                      size="24px"
+                      color="teal"
+                    />
+                  ) : (
+                    "Sign Up"
+                  )}
+                </Button>
+              </form>
+              <Box textAlign="center" mt={6}>
+                <Link fontSize="xs" onClick={switchMode}>
+                  Already with us? Sign In.
+                </Link>
+              </Box>
+            </Box>
+          </Box>
+        ) : (
+          <Box p={2}>
+            {/* Sign In Page */}
+            <Box textAlign="center">
+              <Heading>Welcome Back</Heading>
+            </Box>
+            <Box my={4} textAlign="left">
+              <form onSubmit={handleSubmit}>
+                {error ? <ErrorMessage message={error} /> : null}
+                <FormControl isRequired>
+                  <FormLabel>Email</FormLabel>
+                  <Input
+                    type="email"
+                    placeholder="someone@somewhere.com"
+                    name="email"
+                    onChange={handleChange}
+                  />
+                </FormControl>
+                <FormControl mt={6} isRequired>
+                  <FormLabel>Password</FormLabel>
+                  <Input
```

Comment:
Lets set the value of the input same as state, that way if we want to reset the email and password input, we can do it just by modifying the state

Reply:
ooooh, good catch. I missed that part.